core/services/ocr2/plugins/ocr2vrf/coordinator/block_cache.go

```diff
@@ -0,0 +1,122 @@
+package coordinator
+
+import (
+	"runtime"
+	"sync"
+	"time"
+
+	"github.com/ethereum/go-ethereum/common"
+)
+
+// blockCache is a caching strucuture that allows items to be stored and then evicted
+// based on an eviction window. In this package, it is being used to track in-flight
+// items the coordinator includes in OCR reports, such that the items can be checked against
+// the cache to avoid double-transmissions.
+type blockCache[T any] struct {
+	blockEvictionWindow int64
```

Comment:
Is this eviction window in terms of seconds or in terms of blocks? If it's in seconds, we should make it clear by using `time.Duration` rather than `int64`. If it's blocks, good to keep it at `int64`.

Reply:
I've made the window in `time.Duration` based on this comment, but there is a bit more outside conversion now, as `time.Duration` is meant to be in nanoseconds.